app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :expenses, foreign_key: :author_id
+  validates :name, presence: true
+  # Include default devise modules. Others available are:
+  #:lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :confirmable,
+         :recoverable, :rememberable, :validatable
+end
```

Comment:
- [ ] Great job so far. However, I am unable to confirm my email after the registration which would have given me access to proceed testing your application. I got an error message after entering my data on the register page.  I was supposed to have received the confirmation link either from the page or console, but I got this error message instead. Kindly look into it and fix. 

### Error from browser
![confirmable-error](https://github.com/naanmohammed/Budget-App/assets/26597432/c87861ee-783c-4c94-af4f-b58b1bbb4a78)

### Error from console
![error-from-console](https://github.com/naanmohammed/Budget-App/assets/26597432/cfbe739c-3323-44a2-9c8b-9fadb6ff9c41)